src/components/typhography/heading/styles.ts

```diff
@@ -0,0 +1,67 @@
+import stitches from '../../../stitches.config';
+const { styled } = stitches;
+
+const defaultSettingHeading = {
+  margin: 0,
+  padding: 0,
+  variants: {
+    color: {
+      primary: { color: '$neutral8' },
+      secondary: { color: '$neutral7' },
+      'dark-primary': { color: '$neutral1' },
+      'dark-secondary': { color: '$neutral3' },
+    },
+    weight: {
+      medium: { fontWeight: '400' },
+      bold: { fontWeight: '600' },
+    },
+  },
+};
+
+export const H1 = styled('h1', {
+  ...defaultSettingHeading,
+  variants: {
+    ...defaultSettingHeading.variants,
+    size: {
+      small: { fontSize: '24px', lineHeight: '32px' },
+      medium: { fontSize: '32px', lineHeight: '40px' },
+      normal: { fontSize: '40px', lineHeight: '48px' },
+    },
+  },
+});
+
+export const H2 = styled('h2', {
+  ...defaultSettingHeading,
+  variants: {
+    ...defaultSettingHeading.variants,
+    size: {
+      small: { fontSize: '20px', lineHeight: '28px' },
+      medium: { fontSize: '24px', lineHeight: '32px' },
+      normal: { fontSize: '32px', lineHeight: '40px' },
+    },
+  },
+});
+
+export const H3 = styled('h3', {
+  ...defaultSettingHeading,
+  variants: {
+    ...defaultSettingHeading.variants,
+    size: {
+      small: { fontSize: '16px', lineHeight: '24px' },
+      medium: { fontSize: '20px', lineHeight: '28px' },
+      normal: { fontSize: '24px', lineHeight: '32px' },
+    },
+  },
+});
+
+export const H4 = styled('h4', {
```

Comment:
I think that's necessary can includes a similar const to add 'H5' variant, because it's exist at Design System.

Reply:
I checked with the designer boys and the "h5" does not enter the scope of the heading, the heading goes from h1 to h4